test/y2storage/guided_proposal_test.rb

```diff
@@ -140,7 +140,7 @@
     end
 
     context "when installing in a DM RAID" do
-      let(:scenario) { "empty-dm_raids.xml" }
+      let(:scenario) { "empty-dm_raids_no_sda.xml" }
```

Comment:
Since the test seems to be there just to check than the partitions are correctly created and so on, wouldn't have been easier to restrict the proposal scope than creating a new xml file? Maintaining all those devicegraph definition is a PITA.

Something like this (I haven't tested it) could save us from adding just another blob:
```ruby
 before do
  settings.candidate_devices = ["/dev/mapper/isw_ddgdcbibhd_test1", "/dev/mapper/isw_ddgdcbibhd_test2"]
end
```

So sda is out in a more diplomatic way than killing it. :-)

Reply:
I find this even harder to understand. That's a setup mixture of xml files and code in rspec. This entangles unrelated test cases even more.